### DISCUSSION TITLE
posts/1.-%EC%98%A4%ED%86%A0%EB%A6%AC%EC%8A%B5(Autolisp)-%EC%8B%9C%EC%9E%91%ED%95%98%EA%B8%B0/

### DISCUSSION THREAD

**Discussion by giscus** [IsaacGoCoder/IsaacGoCoder.github.io#4] [General]
# posts/1.-%EC%98%A4%ED%86%A0%EB%A6%AC%EC%8A%B5(Autolisp)-%EC%8B%9C%EC%9E%91%ED%95%98%EA%B8%B0/

오토리습(Autolisp)이란? 오토리습은 오토캐드에서 사용할 수 있는 프로그래밍 언어입니다. 오토 캐드(CAD) 내부의 도형과 도면 뿐만 아니라 외부 프로그램과 연동하여 사용자가 원하는 다양한 작업을 명령어 하나로 수행할 수 있습니다.

https://isaacgocoder.github.io/posts/1.-%EC%98%A4%ED%86%A0%EB%A6%AC%EC%8A%B5(Autolisp)-%EC%8B%9C%EC%9E%91%ED%95%98%EA%B8%B0/

**Comment by yeonmech** [comment]
잘 읽겠습니다 감사합니다

**Reply by GoCoder7** [reply]
안녕하세요
11월중에 새 [블로그](https://blog.gocoder.xyz)로 이전할 예정입니다
방문 해 주셔서 감사합니다~
